src/Ocelot/Configuration/Validator/DownstreamPathTemplateDoesntStartWithForwardSlash.cs

```diff
@@ -1,12 +0,0 @@
-using Ocelot.Errors;
-
-namespace Ocelot.Configuration.Validator
-{
-    public class PathTemplateDoesntStartWithForwardSlash : Error
-    {
-        public PathTemplateDoesntStartWithForwardSlash(string message) 
```

Comment:
Do we also need to delete the enum?

Reply:
delete the enum is good, but consider the people who already use the ocelot, i think just Obsolete it  and remove it in a few next version ? but what can do if people who dependent  on the absolutely value of the  enum ?